src/lib/gadgets/elliptic-curve.ts

```diff
@@ -194,6 +194,32 @@ function verifyEcdsa(
   Provable.assertEqual(Field3.provable, Rx, r);
 }
 
+/**
+ * Bigint implementation of ECDSA verify
+ */
+function verifyEcdsaConstant(
```

Comment:
I just moved this code here from the bottom, because there was too much other stuff getting in between